2019/d01p1FuelCounterUpper.d

```diff
@@ -0,0 +1,7 @@
+unittest {
+  import std.algorithm;
+
+  auto input = [12, 14, 1969, 100756];
```

Comment:
This line establishes `input` as a list of numbers. The `auto` type specifier says "work it out for me" but really it is just less typing than `int[]` :smile:

I've written both C and C++ oldskool as well as lots of modern scripts so this syntax is both familiar and refreshing.